org.jacoco.doc/docroot/doc/license.html

```diff
@@ -275,6 +275,14 @@
       of your accepting any such warranty or additional liability.
 </pre>
 
+<h3>args4j, Version 2.0.28</h3>
+
+<p>
+  <a href="http://args4j.kohsuke.org/">args4j</a> is subject to the terms and
+  conditions of
+  <a href="http://www.opensource.org/licenses/mit-license.php">the MIT license</a>. 
```

Comment:
@Godin I already removed the version. The remaining license issues we clarify in #539.